mimicpy/core/_qmhelper.py

```diff
@@ -32,6 +35,40 @@ def index(qmids, name, space_len=6, col_len=15):
     
     return index
 
+def check_mdp(mdp):
+    nsteps = 1000
```

Comment:
Why static and not as arguments with default values?

Reply:
prepare.getInp() will pass an MDP script to this function (this MDP script is passed to getInp() by the user). This function will check for errors in the MDP script wrt MiMiC (like constraints should not be there, etc.). It then takes the nsteps and dt parameters from the script, converts them to a.u., and return it. This is so that getInp() can use it in the CPMD script.

Sometime, the script may not have nsteps or d or both. In those cases, the function will return the default values defined in the start of the function (those are not static variables though). If the user passed no MDP script (it is optional), then check_mdp get mdp=None. So, it doesn't do any check for errors and just returns the default nsteps, dt.

Reply:
This would be my suggestion
```suggestion
    def check_mdp(mdp, nsteps=1000, dt=0.0001):
```

Reply:
Ok, incorporated.